lib/Client.js

```diff
@@ -3487,22 +3487,30 @@ class Client extends EventEmitter {
      * Get a member that is part of a thread channel
      * @arg {String} channelID The ID of the thread channel
      * @arg {String} userID The ID of the user
-     * @arg {Boolean} withMember Whether to include the guild member object within the response
+     * @arg {Boolean} [withMember] Whether to include the guild member object within the response
      * @returns {Promise<ThreadMember>}
      */
     getThreadMember(channelID, userID, withMember) {
-        return this.requestHandler.request("GET", Endpoints.THREAD_MEMBER(channelID, userID), {
+        return this.requestHandler.request("GET", Endpoints.THREAD_MEMBER(channelID, userID), true, {
             with_member: withMember
         }).then((threadMember) => new ThreadMember(threadMember, this));
     }
 
     /**
      * Get a list of members that are part of a thread channel
      * @arg {String} channelID The ID of the thread channel
+     * @arg {Object} [options] Options for the request
+     * @arg {String} [options.after] Get members after this user ID
+     * @arg {Number} [options.limit=100] The maximum number of thread members to fetch (1-100)
+     * @arg {Boolean} [options.withMember] Whether to include the guild member object for each member of the thread
      * @returns {Promise<Array<ThreadMember>>}
      */
-    getThreadMembers(channelID) {
-        return this.requestHandler.request("GET", Endpoints.THREAD_MEMBERS(channelID), true).then((members) => members.map((member) => new ThreadMember(member, this)));
+    getThreadMembers(channelID, options) {
```

Comment:
```suggestion
    getThreadMembers(channelID, options = {}) {
```
Otherwise will error if we don't pass `options`